services/QuillLMS/db/migrate/20220505155016_create_student_response.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class CreateStudentResponse < ActiveRecord::Migration[5.1]
+  def change
+    create_table :student_responses do |t|
+      t.references :activity_session, null: false, foreign_key: true
+      t.integer :attempt_number
+      t.boolean :correct, null: false
+      t.references :question
+      t.integer :question_number
+      t.float :question_score
+      t.references :student_response_answer_text, foreign_key: true
```

Comment:
Do we have any concerns that these foreign keys will cause performance issues on this giant table. FK's do slow down inserts and they often removed from very, very large tables for performance reasons. Probably worth doing a little bit of research on this and/or keeping an eye on this if you do keep the foreign keys.